Documentation/Migration/Index.rst

```diff
@@ -174,6 +200,121 @@ of typing a long :bash:`docker run...` or :bash:`podman run...` command:
 
     make docs
 
+You should see something like this
+
+.. code-block:: text
+
+    Successfully placed 7 rendered HTML, SINGLEPAGE, and INTERLINK files into /project/Documentation-GENERATED-temp
+
+..  _migrate-possible-errors:
+
+Possible errors using `make docs`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We provide four example errors to guide you through the fixes.
+
+..  _migrate-interlink-inventory-not-found:
+
+Interlink inventory not found: HTTP/2 404
+"""""""""""""""""""""""""""""""""""""""""
+
+.. code-block:: text
+
+    [2024-03-13T12:22:50.661532+00:00] app.WARNING: Interlink inventory not found: HTTP/2 404
+    returned for "https://docs.typo3.org/m/typo3/book-extbasefluid/11.5/en-us/objects.inv.json". [] []
+
+Here you see that the link `https://docs.typo3.org/m/typo3/book-extbasefluid/11.5/en-us` is not found (404 page).
+We can now check via google if there is another link to book Extbasefluid. We found this site `https://docs.typo3.org/m/typo3/book-extbasefluid/10.4/en-us/`
+We can find the hint: `This manual is no longer being maintained for TYPO3 versions 11.5 and above.`. This tells us that the developers stopped to update the
+documentation. We can for example link to the last existing version. Which is 10.4. To do this we have to change the :file:`guides.xml`. Search for the
+
+.. code-block:: xml
+
+    <inventory id="t3extbasebook"
+           url="https://docs.typo3.org/m/typo3/book-extbasefluid/11.5/en-us/"
+    />
+
+and change `11.5` to `10.4`. Generate files again.
+
+
+..  _migrate-inventory-link-with-key-not-found:
+
+Inventory link with key ... not found
+"""""""""""""""""""""""""""""""""""""
+
+.. code-block:: text
+
+    [2024-03-13T12:26:40.940930+00:00] app.WARNING: Inventory link with key "h2document:rest-common-pitfalls"
+    (rest-common-pitfalls) not found.  {"rst-file":"GeneratedExtension/Index","type":"ref","targetRef
+
+We see already that we have to go to file :file:`GeneratedExtension/Index` in the directory "Documentation".
+In there we have to delete the line which contains
+
+.. code-block:: text
+
+    * :ref:`h2document:rest-common-pitfalls`
+
+..  _migrate-nested-php-domain-components-not-supported:
+
+Nested PHP domain components (php:class, php:interface, php:enum etc) are not supported
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+.. code-block:: text
+
+    [2024-03-25T13:26:11.600367+00:00] app.WARNING: Nested PHP domain components
+    (php:class, php:interface, php:enum etc) are not supported.
+    Found php:\Vendor\MyExtension\Interfaces\RequireJsModuleInterface inside
+    \Vendor\MyExtension\Interfaces\AnotherImportantInterface {"rst-file":"Developer/Index.rst"} []
+
+The file :file:`Index.rst` in Documentation/Developer has a wrong indentation. A class must not belong to another class.
```

Comment:
```suggestion
The file :file:`Index.rst` in :file:`Documentation/Developer/` has a wrong indentation. A class must not belong to another class.
```

We always append a slash at the end of folders to distinguish between folders and files.